tests/main.test.ts

```diff
@@ -599,7 +600,7 @@ const maxConcurrentDefaults = {
 
 function createContext(
   issue: Record<string, unknown>,
-  sender: Record<string, unknown>,
+  sender: Record<string, unknown> | undefined,
```

Comment:
Undefined seems impossible?

Reply:
It seems that it can theoretically happen on a `pull_request.modified` payload? Maybe if it is modified by some automation.

Reply:
Seems wrong doesn't make sense to me

Reply:
That's how GitHub code is written, not much I can do:
<img width="704" alt="Screenshot 2024-09-24 at 08 03 44" src="https://github.com/user-attachments/assets/01664fd6-037a-4c6b-aae2-351ae081f8b1">